src/client/table_client.rs

```diff
@@ -1986,6 +2012,20 @@ impl ObTableClientQueryImpl {
             }
         }
 
+        // defense for multiple partition aggreagtion
+        if self.is_aggregation() {
+            if partition_table.len() > 1 {
+                error!(
+                    "do not support aggregation of multiple partitions"
+                );
+                return Err(CommonErr(
+                    CommonErrCode::ObException(OB_NOT_SUPPORTED),
```

Comment:
We should not use the Error Code from the observer, try using other errors.

Reply:
had use the err code from client